app/views/partials/loggedOutErrorMessage.scala.html

```diff
@@ -0,0 +1,14 @@
+@()
+<div id="logged-out-error" class="govuk-error-summary logged-out-error" hidden aria-labelledby="logged-out-error-title"
+role="alert" tabindex="-1" data-module="govuk-error-summary">
+  <h2 class="govuk-error-summary__title logged-out-title" id="logged-out-error-title">
+    You have been signed out due to inactivity
+  </h2>
+  <div class="govuk-error-summary__body logged-out-body">
+    <ul class="govuk-list govuk-error-summary__list">
+      <li>
+        <p><a class="logged-out-error-link">Sign in</a></p>
```

Comment:
No. Because the sign in url changes each time, Sam added code to the [error handling js file](https://github.com/nationalarchives/tdr-transfer-frontend/blob/5a35e485c33d667b7e651575a9496ea8b55968ea/npm/src/errorhandling/index.ts#L110) that sets it